types.js

```diff
@@ -32,11 +32,14 @@ export type SJColor = {|
 // This just prevents some mistakes.
 // $Values will make these unnecessary https://github.com/facebook/flow/issues/627
 type BorderPositionEnum = 0 | 1 | 2 | 3;
+type BorderLineCapStyle = 0 | 1 | 2;
+type BorderLineJoinStyle = 0 | 1 | 2;
 type FillTypeEnum = 0 | 1 | 4 | 5;
-type PatternFillTypeEnum = 0 | 1;
+export type SJPatternFillTypeEnum = 0 | 1 | 2 | 3;
```

Comment:
I think we should avoid exporting these for now, since they are coupled to the sketch-constants enums defined elsewhere and we don't document them in this file.

I think it's worth considering merging them into sketch-constants or maybe depending on sketch-constants and reexporting the constants within our namespace.